sections/errorhandling/usematurelogger.md

```diff
@@ -9,38 +9,78 @@ We all love console.log but obviously, a reputable and persistent logger like [W
 
 ### Code Example – Winston Logger in action
 
-```javascript
+<details>
+<summary><strong>Javascript</strong></summary>
+<pre lang="javascript">
 // your centralized logger object
 var logger = new winston.Logger({
   level: 'info',
   transports: [
     new (winston.transports.Console)()
   ]
 });
-
+<br/>
 // custom code somewhere using the logger
 logger.log('info', 'Test Log Message with some parameter %s', 'some parameter', { anything: 'This is metadata' });
+</pre>
+</details>
 
-```
+<details>
+<summary><strong>Typescript</strong></summary>
+<pre lang="typescript">
+// your centralized logger object
+const logger = new winston.Logger({

```

Comment:
Also unify code examples here?

Reply:
Which parts would you unify here? The two code examples into one? I believe these sections were intended in the beginning, but I'm not completely sure here, maybe ask @goldbergyoni about this

Reply:
Yes, I would only use one example for both languages since there's not much difference